doc/general/general.md

```diff
@@ -542,7 +542,7 @@ With [`<basicHttpBinding>`](https://msdn.microsoft.com/en-us/library/system.serv
 
 **C# call for signing with SOAP:**
 ```cs
-//create proxy and establish authentification
+//create proxy and establish authentication
```

Comment:
I am not really sure... do you think that is true? I love the word authentification... authentication is not cool.. let´s use the old one.. what do you think @FlorianStadlberger ?